### PR TITLE
Swap pairs

### PR DESCRIPTION
**Pull request by jtlap**
adding tests changing names and more bit functions

### CODE REVIEW

File: include/eve/module/core/regular/byte_swap_pairs.hpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+//!   @endcode
+//!
+//!   @groupheader{Callable Signatures}
+//!
+//!   @code
+//!   namespace eve
+//!   {
+//!      template<unsigned_value T, std::ptrdiff_t I0, std::ptrdiff_t I1>
+//!      T byte_swap_pairs(T x, , index_t<I0> const & i0, index_t<I1> const & i1) noexcept;
+//!   @endcode
```
Comment (jfalcou) [inline include/eve/module/core/regular/byte_swap_pairs.hpp:33]: Missing }

File: test/doc/core/regular/bit_reverse.cpp [hunk 2]
```diff
@@ -3,25 +3,31 @@
...
 {
   wide_it pi = {-14, 1, 3, 0};
 
   std::cout << "---- simd" << '\n'
-            << "<- pi              = " << pi << '\n'
-            << "-> bit_reverse(pi)  = " << eve::bit_reverse(pi) << '\n'
-            << "-> sbit_reverse(pi) = " << eve::detail::map(eve::bit_reverse, pi)<< '\n'
+            << "<- pi                         = " << pi << '\n'
+            << "-> bit_reverse(pi)            = " << eve::bit_reverse(pi) << '\n'
+            << "-> sbit_reverse(pi)           = " << eve::detail::map(eve::bit_reverse, pi)<< '\n'
```
Comment (jfalcou) [inline test/doc/core/regular/bit_reverse.cpp:15]: `eve::detail::map(eve::bit_reverse, pi)` should not be here

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: test/doc/core/regular/bit_reverse.cpp (hunk 2 above)
Comment (jtlap) [reply test/doc/core/regular/bit_reverse.cpp]: done

File: include/eve/module/core/regular/byte_swap_pairs.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/core/regular/byte_swap_pairs.hpp:33]: done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**